apps/frontend/core-dapp/src/features/portfolio/PortfolioStore.ts

```diff
@@ -147,6 +147,14 @@ export class PortfolioStore {
     }
   }
 
+  get loadingPositions(): boolean {
```

Comment:
```suggestion
  get isLoadingPositions(): boolean {
```
to distinguish from similar non-bool getters like `tradingPositions`

Reply:
good suggestion!